src/adagrams.js

```diff
@@ -1,8 +1,112 @@
-const Adagrams = {
+import { tsConstructorType } from "@babel/types";
+
+class Adagrams {
+
+  // Draw letters for a game of Adagrams.
   drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+    const constLetterPool = ['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'B', 'B', 'C', 'C', 'D', 'D', 'D', 'D', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'F', 'F', 'G', 'G', 'G', 'H', 'H', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'J', 'K', 'L', 'L', 'L', 'L', 'M', 'M', 'N', 'N', 'N', 'N', 'N', 'N', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'P', 'P', 'Q', 'R', 'R', 'R', 'R', 'R', 'R', 'S', 'S', 'S', 'S', 'T', 'T', 'T', 'T', 'T', 'T', 'U', 'U', 'U', 'U', 'V', 'V', 'W', 'W', 'X', 'Y', 'Y', 'Z'];
+    let playerHand = [];
+
+    for (let i = 0; i < 10; i++) {
+      const randomNum = Math.floor(Math.random() * (constLetterPool.length + 1));
+      playerHand.push(constLetterPool[randomNum]);
+    };
+
+    return playerHand;
+  }
+
+  // Check that the inputted word can be made from the letters in hand.
+  usesAvailableLetters(input, lettersInHand) {
+    const inputArray = input.toUpperCase().split('');
+    let checkAvail = {};
+
+    for (const letter of lettersInHand) {
+      if (checkAvail[letter] === undefined) {
+        checkAvail[letter] = 1;
+      } else {
+        checkAvail[letter] += 1;
+      };
+    };
+
+    for (const letter of inputArray) {
+      if (checkAvail[letter] !== undefined) {
+        checkAvail[letter] -= 1;
+      } else {
+        checkAvail[letter] = -1;
+      };
+    };
+
+    for (const count of Object.values(checkAvail)) {
+      if (count < 0) return false;
+    };
+
+    return true;
+  }
+
+  // Score the word.
+  scoreWord(word) {
+    const wordArray = word.toUpperCase().split('');
+    let total = 0;
+
+    const scoringTable = {
+      1: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'],
+      2: ['D', 'G'],
+      3: ['B', 'C', 'M', 'P'],
+      4: ['F', 'H', 'V', 'W', 'Y'],
+      5: ['K'],
+      8: ['J', 'X'],
+      10: ['Q', 'Z']
+    };
+
+    for (const letter of wordArray) {
+      for (const scoreLetter of Object.keys(scoringTable)) {
+        if (scoringTable[scoreLetter].includes(letter)) total += Number(scoreLetter);
```

Comment:
While single line conditionals permit you to drop the `{}`, to increase consistency and readability, it is best practice to include the `{}`